### PR TITLE
Feature/fonts

### PR DESCRIPTION
**Pull request by NicholasLYang**
Added Old English MT to the masthead

### CODE REVIEW

File: src/js/modules/main/components/Navbar.js [hunk 1]
```diff
@@ -33,9 +28,12 @@ const styles = {
     margin: "0 20px 0 25px"
   },
   masthead: {
-    width: "10em",
-    padding: "5px 10px 5px 10px",
-    backgroundColor: "white"
+    color: "White",
```
Comment (squirrelsquirrel78) [inline src/js/modules/main/components/Navbar.js]: lowercase

File: src/js/modules/main/components/Navbar.js [hunk 2]
```diff
@@ -46,10 +44,15 @@ const Navbar = ({ classes,
   <Toolbar style={styles.toolbar}>
     <ToolbarGroup key={1}>
       <IconButton onClick={openNavDrawer}>
-        <NavigationMenu color={blue50} />
+        <NavigationMenu color={"White"} />
```
Comment (squirrelsquirrel78) [inline src/js/modules/main/components/Navbar.js:47]: lowercase

File: src/js/modules/main/components/Navbar.js [hunk 3]
```diff
@@ -46,10 +44,15 @@ const Navbar = ({ classes,
...
-      <Link to="/" className={classes.toolbarTitle}>
-        <ReactSVG className={classes.masthead} path="/img/masthead.svg" />
+      <Link
+        to="/"
+        className={classes.mastheadLink}
+      >
+        <div className={classes.masthead}>
+          The Spectator
+        </div>
       </Link>
```
Comment (squirrelsquirrel78) [inline src/js/modules/main/components/Navbar.js]: can you call these `brandingLink` and `branding`? (1) That's how I do it in CFA and (2) Masthead is what I call the condensed-nav component in CFA.

**Review by squirrelsquirrel78 — CHANGES_REQUESTED** [review]
Some naming beans

File: src/js/modules/main/components/Navbar.js (hunk 3 above)
Comment (NicholasLYang) [reply src/js/modules/main/components/Navbar.js]: Why branding? I'm fine either way but I'm not sure it's the best name

File: src/js/modules/main/components/Navbar.js (hunk 1 above)
Comment (NicholasLYang) [reply src/js/modules/main/components/Navbar.js]: Lowercase didn't work for some weird ass reason. We're talking about the White, right?

File: src/js/modules/main/components/Navbar.js (hunk 3 above)
Comment (squirrelsquirrel78) [reply src/js/modules/main/components/Navbar.js]: Because we're branding the navbar with the spec logo.

**Comment by NicholasLYang** [comment]
Eh, that's tenuous. Branding implies like advertisement. SpecLogo or
SpecHeader is okay

**Pull request merged**